src/LazyHashMap.php

```diff
@@ -25,29 +25,21 @@ public function __construct(callable $callback)
 
 
 	/**
+	 * @param  string $key
 	 * @return mixed
 	 */
-	protected function load(string $key)
+	public function __get(string $key)
 	{
-		return call_user_func($this->callback, $key);
+		return $this->$key = $this->load($key);
 	}
 
-}
 
 
-/**
- * @internal
- */
-final class LazyHashMap extends LazyHashMapBase
-{
-
 	/**
-	 * @param  string $key
 	 * @return mixed
 	 */
-	public function __get(string $key)
+	protected function load(string $key)
```

Comment:
No reason to make it protected when the class is final and the inheritance was removed.